react/README.md

```diff
@@ -0,0 +1,218 @@
+# Airbnb React/JSX Style Guide
+
+*A mostly reasonable approach to React and JSX*
+
+## Table of Contents
+
+  1. [Basic Rules](#basic-rules)
+  1. [Naming](#naming)
+  1. [Declaration](#declaration)
+  1. [Alignment](#alignment)
+  1. [Quotes](#quotes)
+  1. [Spacing](#spacing)
+  1. [Props](#props)
+  1. [Parentheses](#parentheses)
+  1. [Tags](#tags)
+  1. [Ordering](#ordering)
+
+## Basic Rules
+
+  - Only include one React component per file.
+  - Always use JSX syntax.
+  - Do not use `React.createElement` unless you're initializing the app from a `.js` file.
+
+## Naming
+
+  - **Extensions**: Use `.jsx` extension for React components.
+  - **Filename**: Use PascalCase for filenames. E.g., `ReservationCard.jsx`.
+  - **Reference Naming**: Use PascalCase for React components and camelCase for their instances:
+    ```javascript
+    // bad
+    const reservationCard = require('./ReservationCard');
+
+    // good
+    const ReservationCard = require('./ReservationCard');
+
+    // bad
+    const ReservationItem = <ReservationCard />;
+
+    // good
+    const reservationItem = <ReservationCard />;
+    ```
+
+    **Component Naming**: Use the filename as the component name. So `ReservationCard.jsx` should have a reference name of ReservationCard. However, for root components of a directory, use index.jsx as the filename and use the directory name as the component name:
+    ```javascript
+    // bad
+    const Footer = require('./Footer/Footer.jsx')
+
+    // bad
+    const Footer = require('./Footer/index.jsx')
+
+    // good
+    const Footer = require('./Footer')
+    ```
+
+
+## Declaration
+  - Do not use displayName for naming components, instead name the component by reference.
+
+    ```javascript
+    // bad
+    export default React.createClass({
+      displayName: 'ReservationCard',
+      // stuff goes here
+    });
+
+    // good
+    const ReservationCard = React.createClass({
+      // stuff goes here
+    });
+    export default ReservationCard;
+    ```
+
+## Alignment
+  - Follow these alignment styles for JSX syntax
+
+    ```javascript
+    // bad
+    <Foo bar="bar"
+         baz="baz" />
+
+    // good
+    <Foo
```

Comment:
This is for when all the props do not fit in a single line. If it fits in a single line then see line 87 below.